plugins/src/webrtcsink/imp.rs

```diff
@@ -538,12 +538,8 @@ impl VideoEncoder {
             _ => unreachable!(),
         }
 
-        let mut s = self
-            .filter
-            .property::<gst::Caps>("caps")
-            .structure(0)
-            .unwrap()
-            .to_owned();
+        let current_caps = self.filter.property::<gst::Caps>("caps");
+        let mut s = current_caps.structure(0).unwrap().to_owned();
```

Comment:
You're optimistic here. The caps could be `None` or empty/any, in either case this would panic :)

Reply:
yes, we're OK here, also the code this replaces was doing that too :)